app/src/main/java/com/example/calculator/activity/CalculatorActivity.kt

```diff
@@ -48,6 +49,24 @@ class CalculatorActivity : AppCompatActivity() {
         }
     }
 
+    override fun onSaveInstanceState(outState: Bundle) {
+        super.onSaveInstanceState(outState)
+        outState.putSerializable("firstOperand", presenter.onSaveInstance().getFirstOperand())
+        outState.putSerializable("secondOperand", presenter.onSaveInstance().getSecondOperand())
+        outState.putSerializable("operand", presenter.onSaveInstance().getOperator())
+    }
+
+    override fun onRestoreInstanceState(savedInstanceState: Bundle) {
+        super.onRestoreInstanceState(savedInstanceState)
+        presenter.onResume(
+            CalculatorEntity(
+                savedInstanceState.get("firstOperand").toString(),
```

Comment:
here use the constants that you will create

Reply:
Same like last comment